testcafe/tests/pages/drive/drive-model-private.js

```diff
@@ -460,4 +461,27 @@ export default class privateDrivePage extends DrivePage {
       .ok('Delete button does not show up')
       .click(selectors.btnModalSecondButton)
   }
+
+  async typeInSearchInput({
+    text: text,
+    screenshotPath: screenshotPath,
+    withMask = false
+  }) {
+    await isExistingAndVisibile(selectors.searchInput, 'selectors.searchInput')
+    await t.typeText(selectors.searchInput, text)
+    //wait 1s for result to appears
+    await t.wait(SEARCH_DELAY)
+    await t.fixtureCtx.vr.takeScreenshotAndUpload({
+      screenshotPath: `${screenshotPath}`,
```

Comment:
```suggestion
      screenshotPath,
```

No need to interpolate the variable here.